common/lib/xmodule/xmodule/templates/problem/jsinput_response.yaml

```diff
@@ -5,67 +5,176 @@ metadata:
+                message += ' Lower endpoint is wrong.'
+              if(guess_upper_closed != True and interval_type[1] == 'closed'):
+                final_grade = final_grade - type_penalty
+                message += ' Upper endpoint is wrong.'
+              if(guess_upper_closed == True and interval_type[1] != 'closed'):
+                final_grade = final_grade - type_penalty
+                message += ' Upper endpoint is wrong.'
+            
+        else:
+          
+          farthest = max(abs(correct_number - guess_upper), abs(correct_number - guess_lower))
+          
+          if farthest &lt; tolerance[0]:
+            final_grade = brackets[0]
+            message = 'Close enough! Actual answer: ' + str(correct_number)
+          elif farthest &lt; tolerance[1]:
+            final_grade = brackets[1]
+            message = 'Close. You are off by ' + str(farthest)
+          elif farthest &lt; tolerance[2]:
+            final_grade = brackets[2]
+            message = 'Not very close. You are off by ' + str(farthest)
+          else:
+            final_grade = brackets[3]
+            message = 'Your range is too large to get points.'
+          
+          if guess_upper &gt; correct_number and guess_lower &lt; correct_number:
+            message += ' The answer is within your range.'
+          else:
+            message += ' The answer is outside your range.'
+          
+        if not feedback:
+          message = ''
+
+        if final_grade &gt; 0.95:
+          isOK = True
+        elif final_grade &gt; 0.05:
+          isOK = "Partial"
+        else:
+          isOK = False
+        
+        return {
+          'input_list': [
+              { 'ok': isOK, 'msg': message, 'grade_decimal': final_grade},
+          ]
+        }
+
+
+
+      </script>
+        <script type="text/javascript">
+      $(document).ready(function(){
+      	console.log('Let\'s go!');
+      });
+
+      // Logs a javascript object.
+      function logThatThing(ThatThing){
+
+      	// Log it to the console just to verify it's working
+      	console.log(JSON.stringify(ThatThing));
+      	
+      	// Send it to the official edX logamajig!
+      	// Logger.log("harvardx.public_demo.range_guesser", ThatThing);
+
+      }
       </script>
-            <p>In the following image, click the objects until the cone is yellow and the cube is blue.</p>
-            <jsinput gradefn="WebGLDemo.getGrade"
-              get_statefn="WebGLDemo.getState"
-              set_statefn="WebGLDemo.setState"
-              initial_state='{"selectedObjects":{"cube":true,"cylinder":false}}'
-              width="400"
-              height="400"
-              html_file="https://studio.edx.org/c4x/edX/DemoX/asset/webGLDemo.html"
-              sop="false"/>
-          </customresponse>
+        <p>What is the range of passengers that can fit on a 747 aircraft? Set the low and high bounds. (416 and 660 persons)</p>
+        <p class="sr" aria-hidden="true">
```

Comment:
Is this "sr" block here the a11y you are demonstrating? Is there anything else about this particular problem that demonstrates a11y concepts (I'd like to know in case we want to reduce the complexity of this problem).